data/timesketch.conf

```diff
@@ -26,13 +26,13 @@ SQLALCHEMY_DATABASE_URI = 'postgresql://<USERNAME>:<PASSWORD>@localhost/timesket
 # Make sure that the Elasticsearch server is properly secured and not accessible
 # from the internet. See the following link for more information:
 # http://www.elasticsearch.org/blog/scripting-security/
-ELASTIC_HOST = '127.0.0.1'
```

Comment:
If we remove that now, would that break peoples installation if they simply update or would they get an error message if they still have the old values?

Reply:
They get an error message.